### DISCUSSION TITLE
can it be installed on ubuntu server？

### DISCUSSION THREAD

**Discussion by wcy168** [rustdesk/rustdesk#8115] [General]
As the title suggests

**Comment by xlionjuan** [comment]
Client: No, because you need GUI

**Reply by serkansmg** [reply]
but teamviewer can run on ubuntu server without gui. 
forexample i can run teamviewer on arm64 without any gui, no gnome, xcfe etc..  just with a dummy driver. 

here is the bash code:
sudo su
apt-get install xserver-xorg-video-dummy -y
tee /etc/X11/xorg.conf<<EOF 
Section "Device"
    Identifier  "Configured Video Device"
    Driver      "dummy"
    # Default is 4MiB, this sets it to 16MiB
    VideoRam    16384
EndSection

Section "Monitor"
    Identifier  "Configured Monitor"
    HorizSync 31.5-48.5
    VertRefresh 50-70
EndSection

Section "Screen"
    Identifier  "Default Screen"
    Monitor     "Configured Monitor"
    Device      "Configured Video Device"
    DefaultDepth 24
    SubSection "Display"
    Depth 24
    Modes "1024x800"
    EndSubSection
EndSection
EOF

wget https://download.teamviewer.com/download/linux/teamviewer-host_arm64.deb
echo "installing teamviewer"
apt install ./teamviewer-host_arm64.deb
teamviewer setup
teamviewer passwd yourpassword
teamviewer info 


with this method , i can run teamviewer on raspberry pi, asus tinkerboard and any other linux device just replacing arm64 with the correct cpu type

so why does not rust do it?

**Reply by xlionjuan** [reply]
I know TeamViewer can able show tty, but RustDesk can't
